pkgbuild/Resources/preinstall

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
/usr/bin/env bash if the bash is somewhere else.

Reply:
updated